Tests/Clappr_Tests/Classes/Plugin/Core/MediaControl/QuickSeekMediaControlPluginTests.swift

```diff
@@ -79,6 +83,30 @@ class QuickSeekMediaControlPluginTests: QuickSpec {
                     }
                 }
                 
+                context("and there are overlay plugins") {
+                    it("ignores them and seeks") {
+                        mediaControl.view.layoutIfNeeded()
+                        overlayPlugin.view.layoutIfNeeded()
+                        playButton.view.layoutIfNeeded()
+                        
+                        let shouldSeek = quickSeekPlugin.shouldSeek(point: CGPoint(x: 260, y: 100))
```

Comment:
@BruunoMM, I was wondering if it would make sense to name this point (and other points created in this file) using a constant or, if the point belongs to a specific plugin, use a syntax like `plugin.view.center.x`, for example. I think that naming it would provide more context and therefore readability to the test, but I'm not sure if this is relevant enough, though. 

What do you think?

Reply:
That is a cool suggestion. It would really improve the readability of the test.
Yes, can do! o/